.github/workflows/ui-automation.yml

```
@@ -41,18 +41,18 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3.1.0          
+      - uses: actions/checkout@v3.1.0
       - uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)